index.html

```diff
@@ -116,7 +116,7 @@ <h1>UE300160 dashboard</h1>
             <div class="col-3">
                 <div class="list-group">
                     <a href="https://github.com/alinchenxt" class="list-group-item list-group-item-action active"> 🐑 Alina (alinchenxt) </a>
-                    <a href="git@github.com:alinchenxt/python_hw.git" class="list-group-item list-group-item-action"> 🐍 Python homework </a>
+                    <a href="https://github.com/alinchenxt/python_hw.git" class="list-group-item list-group-item-action"> 🐍 Python homework </a>
```

Comment:
the .git is technically not necessary, but the URL resolves correctly, and that's all that matters :D